t/to_words/to_words_ubi_8.4.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package               : to_words
+# Version               : v1.0.1
+# Source repo           : https://github.com/taimur-akhtar/to_words.git
+# Tested on             : UBI 8.4
+# Language              : Ruby
+# Travis-Check          : True
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Vikas . <kumar.vikas@in.ibm.com>
+#
+# Disclaimer            : This script has been tested in root mode on given
+# ==========              platform using the mentioned version of the package.
+#                         It may not work as expected with newer versions of the
+#                         package and/or distribution. In such case, please
+#                         contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+export PACKAGE_NAME=to_words
+export PACKAGE_URL=https://github.com/taimur-akhtar/to_words.git
+
+if [ -z "$1" ]; then
+  export PACKAGE_VERSION=v1.0.1
+else
+  export PACKAGE_VERSION=$1
+fi
+if [ -d "${PACKAGE_NAME}" ] ; then
+  rm -rf ${PACKAGE_NAME}
+fi
+
+
+yum install -y git ruby procps yum-utils wget
+
+yum-config-manager --add-repo http://mirror.centos.org/centos/8/AppStream/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/PowerTools/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/virt/ppc64le/ovirt-44/
+
+wget https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official && mv RPM-GPG-KEY-CentOS-Official /etc/pki/rpm-gpg/. && rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
```

Comment:
I am not sure this will work now and in case we should not enable unwanted repositories , please update